mongodb-datastore/db/mongodb_event_repo.go

```diff
@@ -146,7 +146,7 @@ func (mr *MongoDBEventRepo) DropProjectCollections(event keptnapi.KeptnContextEx
 	const dropCollectionErrorMsg = "failed to drop collection %s: %v"
 	if err != nil {
 		err := fmt.Errorf(dropCollectionErrorMsg, projectCollection.Name(), err)
-		logger.WithError(err).Error("Could not drop project collection.")
+		logger.Errorf("Could not drop project collection: %v", err)
```

Comment:
I don't think we need to build the error message again, in this case it will be:

Could not drop project collection: failed to drop collection: <name>: <error>

I would suggest something like this instead:

`logger.Errorf("Could not drop project collection '%s': %v",projectCollection.Name(), err)`